<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HackathonDetailView</class>
 <widget class="QWidget" name="HackathonDetailView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Detail HNamackathon</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Date d√©but</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Date de Fin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Sponsor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="adressLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sponsorLineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QDateTimeEdit" name="startDateTimeEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QDateTimeEdit" name="endDateTimeEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="teamListView"/>
   </item>
   <item>
    <widget class="QListView" name="stepListView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
